core/src/main/java/de/jplag/JPlag.java

```diff
@@ -35,43 +37,49 @@ private static Version loadVersion() {
 
     /**
      * Creates and initializes a JPlag instance, parameterized by a set of options.
-     * @deprecated in favor of static {@link #run(JPlagOptions)}.
      * @param options determines the parameterization.
      */
-    @Deprecated(since = "4.3.0")
     public JPlag(JPlagOptions options) {
```

Comment:
Should stay deprecated to tie the object creation to the run call. Just introduce a new run method that overloads the existing one.